Sources/Commands/Completions+bash.swift

```diff
@@ -0,0 +1,105 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Basic
+import Utility
+
+/// Template for Bash completion script.
+///
+/// - Parameter stream: output stream to write the script to.
+func bash_template(on stream: OutputByteStream) {
+    stream <<< "#!/bin/bash\n"
+    
+    stream <<< "_swift() \n"
+    stream <<< "{\n"
+    stream <<< "    declare -a cur prev\n"
+    
+    // Setup `cur` and `prev` variables, these will be used in the various
+    // functions this script will contain. For example if the user requests
+    // completions for `swift pack⇥`, `$cur` is `pack` and `$prev` is `swift`.
+    stream <<< "    cur=\"${COMP_WORDS[COMP_CWORD]}\"\n"
```

Comment:
It was added after this PR, but nevertheless using multi-line strings would be a simplification of this code.